near-sdk/src/store/lazy_option/mod.rs

```diff
@@ -115,7 +115,7 @@ where
     /// be reloaded.
     pub fn get_mut(&mut self) -> &mut Option<T> {
         self.cache.get_or_init(|| load_and_deserialize(&self.storage_key));
-        let entry = self.cache.get_mut().expect("cell should be filled above");
+        let entry = self.cache.get_mut().unwrap_or_else(|| unreachable!());
```

Comment:
this feels odd: I'd expect `.expect` and `.unwrap_or_esle(|| unreachable!())` to be equivalent in terms of code size: both panic with a specific string.

Reply:
hmm, I think I was under the false assumption that unreachable in wasm doesn't call `panic!`. I guess in this case it does save some data from not having new custom strings from being put into the wasm binary.

Perhaps better to just have all these calls just call `sys::panic()` internally to remove file info and the static string in most cases?

Reply:
>hmm, I think I was under the false assumption that unreachable in wasm doesn't call panic!.

were you thinking about `std::arch::wasm32::unreachable`? `sys::panic()` is also fine.